src/components/NewPosition/ConcentrationTypeSwitch/style.ts

```diff
@@ -10,7 +10,7 @@ export const useTabsStyles = makeStyles<{ value: number }>()((_theme, { value })
     width: 160,
     backgroundColor: colors.invariant.black,
 
-    [_theme.breakpoints.down('xs')]: {
```

Comment:
![image](https://github.com/invariant-labs/webapp-a0/assets/102986585/151343f5-9fd2-44a1-89b6-f82424881546)
i would put concentration type switch above price range component

Reply:
![image](https://github.com/invariant-labs/webapp-a0/assets/102986585/02640033-2332-4c6f-9ef3-60d0e6415bbd)
also i think font size of that text should be changed to smaller if it doesn't fit input

Reply:
i hided  overflow for long inputs because similar solution we have on swap page
![image](https://github.com/invariant-labs/webapp-a0/assets/93533892/a809300c-b10b-4307-a30d-0874f713ecb0)

Reply:
looks good